### PR TITLE
Fix #77 and fix scroll to top button to properly scroll to top (on mobile)

### LINKED ISSUE #77: Volunteer page content should be deduplicated
While working on #76, I found that the content for the volunteer tabs was duplicated in multiple places and that the bullet points describing what each team does were out of sync in both. I fixed this, but clearly missed that there were two occurrences in an earlier PR. That said, there really shouldn't be two occurrences in the first place.

So, the aim of this issue is to find a way to consolidate the data on each card so that there is a single source of truth for that data and it is automatically used in both places. (Note that it's better to use templates for this, rather than JavaScript strings containing HTML.)

/cc @hoangviet1993

### PR DESCRIPTION
**Pull request by hoangviet1993**
- [x] Fix #77 : Introduce templates to reduce HTML code duplication
- [x] Fix scroll to top button not properly scrolling (it was scrolling to nav bar but nav bar is now fixed at top of page, instead scroll to top will always scroll to upper most div)
- [x] Other misc lint changes.

Changes are live at :
https://oppia-foundation-test-server.appspot.com/volunteer

### CODE REVIEW

File: app/static/pages/volunteer/Volunteer.js [hunk 1]
```diff
@@ -56,17 +66,22 @@ oppiaFoundationWebsite.controller('VolunteerPage', [
         scrollToTopButton.removeClass('scrolling');
       }
     });
+    $scope.buttonDisabled = false;
```
Comment (seanlip) [inline app/static/pages/volunteer/Volunteer.js]: What's this for -- is it used? Also, from the name, not sure which button it refers to.

File: app/static/pages/volunteer/Volunteer.js [hunk 2]
```diff
@@ -34,6 +34,16 @@ oppiaFoundationWebsite.controller('VolunteerPage', [
       title: 'Marketing',
       templateUrl: '/pages/volunteer/tabs_template/marketing_tab.html'
     }];
+    $scope.volunteerWorkDescription = {
```
Comment (seanlip) [inline app/static/pages/volunteer/Volunteer.js]: $scope.volunteerWorkDescriptionUrls

File: app/static/pages/volunteer/tabs_template/design_tab.html [hunk 3]
```diff
@@ -15,14 +15,7 @@
...
     </p>
-    <p class="oppia-foundation-bold-text">
-      What we do:
-    </p>
-    <ul>
-      <li>Create graphics for lessons</li>
-      <li>Address UX issues that students face</li>
-      <li>Make it easy for lesson creators to get started</li>
-    </ul>
+    <ng-include src="volunteerWorkDescription['design']"></ng-include>
```
Comment (seanlip) [inline app/static/pages/volunteer/tabs_template/design_tab.html]: volunteerWorkDescriptionUrls.design (use dot notation if the key is a string constant).

Ditto elsewhere.

File: app/static/pages/volunteer/work_description_templates/design_description.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```
Comment (seanlip) [inline app/static/pages/volunteer/work_description_templates/design_description.html]: Not sure you need "DOCTYPE html" if this is just a template

File: app/static/pages/volunteer/work_description_templates/design_description.html [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+<p class="oppia-foundation-bold-text">
+    What we do:
```
Comment (seanlip) [inline app/static/pages/volunteer/work_description_templates/design_description.html]: Here and elsewhere: 2 space indentation, not 4

File: app/static/pages/volunteer/volunteer.html [hunk 6]
```diff
@@ -253,21 +233,7 @@ <h3 class="md-display-1 volunteer-page-content-title">
...
-          <li>
-            Work closely with local organizations to ensure that our lessons
-            are useful for their students.
-          </li>
-          <li>
-            Find ways to tell Oppia's story, and inspire others to make a
-            positive difference in the world.
-          </li>
-        </ul>
+        <ng-include src="volunteerWorkDescriptionUrlsUrls.marketing"></ng-include>
```
[the remark this answers is not in the document]
Comment (hoangviet1993) [reply app/static/pages/volunteer/volunteer.html]: My bad here. I did not have a lot of time to careful review my code changes. I have added extra photographic evidence.

**Comment by hoangviet1993** [comment]
Some photos for my changes (no GUI changes are expected)
GIF:

Screenshot of mobile view:
screencapture-localhost-8080-volunteer-2018-10-03-11_35_22

**Comment by hoangviet1993** [comment]
PTAL @seanlip

**Comment by seanlip** [comment]
LGTM. Thanks!

**Pull request merged**